src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  height: 60px;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    margin-right: 50px;
+    list-style-type: none;
+  }
+
+  &__wraper {
+    margin-left: 20px;
+  }
+
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    display: flex;
+    height: 60px;
+    line-height: 60px;
+    transition: 0.5s;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+    text-decoration: none;
+  }
+}
+
+.is-active {
+  text-decoration: none;
+  position: relative;
+  color: #00acdc;
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+  transition: 0.5s;
```

Comment:
the `nav__link` already have a lot of set properties so you can leave just needed here